funds-manager/funds-manager-server/src/handlers.rs

```diff
@@ -129,3 +129,18 @@ pub(crate) async fn withdraw_fee_balance_handler(
 
     Ok(warp::reply::json(&"Fee withdrawal initiated..."))
 }
+
+/// Handler for creating a hot wallet
+pub(crate) async fn create_hot_wallet_handler(
```

Comment:
should we have this handler check against a hot wallet already existing in the DB for the given vault? allowing this may confuse subsequent lookup logic